frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/hooks/useStrategyParameters.tsx

```diff
@@ -5,7 +5,12 @@ import type { FeatureStrategySchema } from 'openapi';
 import { RolloutParameter } from '../RolloutParameter/RolloutParameter';
 
 export const useStrategyParameters = (
-    strategy: Partial<FeatureStrategySchema>,
+    strategy: Partial<
+        Pick<
+            FeatureStrategySchema,
+            'name' | 'constraints' | 'variants' | 'parameters'
+        >
+    >,
```

Comment:
This lets us use this with playground strategy results too.

Reply:
`RolloutParameter` adds `StrategyVariants`. Do we need this in the playground in this spot?